### PR TITLE
use `assign` in `csvToJson` method

### PR DESCRIPTION
**Pull request by EbrahimKreem**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. 
If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to static food data?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

#194

### CODE REVIEW

**Comment by atherdon** [comment]
should i merge it?

>

File: src/csvToJson.js [hunk 1]
```diff
@@ -1,36 +1,19 @@
-/* eslint-disable no-unused-vars */
 import { write, readDir } from '@groceristar/static-data-generator';
-// import { joinPath } from './utils';
-import fs from 'fs';
-import parseCsv from './parseCsv';
 
-function makeFullPath(pathData, filetype) {
```
Comment (atherdon) [inline src/csvToJson.js]: when you removing makeFullPath, did you check if it not called at any places, right?

File: src/csvToJson.js [hunk 2]
```diff
@@ -1,36 +1,19 @@
...
 
-// @TODO update this method later, when we'll migrate to `write` from generator
-
-// @TODO as this method using "generateJsonFiles" method - it should be updated.
-// or maybe move it into generator file, etc.
-const assign = (file, dataEntries) => {
+const assign = async (fileInfo, dataEntries) => {
   // @TODO add if env.development and use console.log(xxx)
-  const maxEntriesPerFile = 10000;
+  const maxEntriesPerFile = 1000;
```
Comment (atherdon) [inline src/csvToJson.js]: I was thinking, maybe we should set it as method attribute as well, so we can tweak it easily

File: src/csvToJson.js (hunk 1 above)
Comment (EbrahimKreem) [reply src/csvToJson.js]: yes

**Pull request merged**